docs/UserGuide.adoc

```diff
@@ -20,39 +20,56 @@ By: `Team W10-B2`      Since: `Aug 2017`      Licence: `MIT`
 === About
 
 This user guide will provide you with instructions on how to use the KayPoh! application.
-You should not need any prior technical knowledge to make use of this guide.
+You should not need any prior technical knowledge to make use of this guide, however,
+some knowledge about a command line interface would be helpful.
 
 === Product Overview
 
-KayPoh! is a desktop contact management application for desktop computers for social users.
+Have you, at some point in time, got curious about what your friends, family, or maybe even
```

Comment:
Splitting at this point sounds a bit unnatural when written
Have you ever gotten curious/ have you ever been curious